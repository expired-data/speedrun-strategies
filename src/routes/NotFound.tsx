import React, { FC } from "react";
import styled from "styled-components";

const CenteredDiv = styled.div`
  margin: auto;
  width: 500px;
  margin-top: 20vh;
  text-align: center;
  font-size: 2rem;
`;

export const NotFound: FC<{}> = () => (
  <CenteredDiv>The page you are looking for does not exist (yet).</CenteredDiv>
);
