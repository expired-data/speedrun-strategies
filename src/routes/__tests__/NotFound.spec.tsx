import React from "react"; 
import { render } from "@testing-library/react";
import { NotFound } from "../NotFound"; 

test("NotFound page renders as expected", () => { 
    const { container } = render(<NotFound />)
    expect(container.innerHTML).toMatchSnapshot();
}); 