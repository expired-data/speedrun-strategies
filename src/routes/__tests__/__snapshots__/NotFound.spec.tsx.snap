// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound page renders as expected 1`] = `"<div class=\\"sc-bdfBwQ jYzGAO\\">The page you are looking for does not exist.</div>"`;
